File: Sprint 3/Digital_Books/src/app/createbook/createbook.model.ts
import{NgForm,FormGroup,Validators,FormBuilder,FormControl} from "@angular/forms";

export class CreateBook {

    title: string = '';
    category: string = '';
    image: string = '';
    price: string = '';
    publisher: string = '';
    active: string = '';
    content: string = '';
    id:string = '';
    author: string = '';
    releasedDate: string = '';


    formCreateBookGroup:FormGroup;
    constructor(){
        var _builder=new FormBuilder();

        this.formCreateBookGroup=_builder.group({});
        this.formCreateBookGroup.addControl("createbooktitle",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookcategory",new FormControl('',Validators.required));
       // this.formCreateBookGroup.addControl("createbookimage",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookprice",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookpublisher",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookactive",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookcontent",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookauthor",new FormControl('',Validators.required));
        this.formCreateBookGroup.addControl("createbookreleasedDate",new FormControl('',Validators.required));
       
    }
}

File: Sprint 3/CustomerAPP/CustomerAPP/Models/createbookmodel.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CustomerAPP.Models
{
    public class createbookmodel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Category { get; set; }
        public IFormFile Image { get; set; }
        public string Price { get; set; }
        public string Publisher { get; set; }
        public string Active { get; set; }
        public string Content { get; set; }
        public string Author { get; set; }
        public string ReleasedDate { get; set; }
    }
}
